it/src/main/java/com/google/cloud/teleport/it/PerformanceBenchmarkingBase.java

```diff
@@ -201,6 +201,7 @@ public boolean waitForNumMessages(String jobId, String pcollectionName, Long exp
   protected Map<String, Double> getMetrics(JobInfo jobInfo, String inputPcollection)
       throws ParseException, InterruptedException, IOException {
     // Metrics take up to 3 minutes to show up
+    LOG.info("Sleeping for 4 minutes to query metrics.");
     Thread.sleep(Duration.ofMinutes(4).toMillis());
```

Comment:
awaitility.org could be an alternative so we keep polling for metrics instead of hardcoding X minutes. but it's ok to ignore for now (or maybe add a TODO with intentions)

Reply:
This is a bit complex since the metrics are available but they are late. Sleeping for 4 minutes guarantees that we get the most accurate metrics. However, I agree that we shouldn't hardcode X minutes.

I will look into this and fix this later. Added a todo to do this.